src/components/Modals/Onboarding/OnboardingReview.vue

```diff
@@ -36,6 +36,16 @@
               ><span> {{ userName.lastName }}</span></label
             >
           </div>
+          <div
+            class="onboarding-inputWrapper onboarding-inputWrapper--name onboarding-inputWrapper--description"
+          >
+            <label class="onboarding-label"
+              ><span>Entrance Season<span class="onboarding-required-star">*</span></span></label
```

Comment:
unrelated to this PR: why bother displaying the * in OnboardingReview? since the review page does not allow you to change data directly, it's sort of irrelevant whether the data you put in was required. also, you need to fill out all required fields before you reach this page anyways.

Reply:
Yeah you make a good point! We didn't always force people to fill in required fields before reaching the review page so it might just be leftover from then. Feel free to bring this up to a designer (maybe Yuxuan) and if they agree we can axe it 😄